task/minio/0.1/README.md

```diff
@@ -0,0 +1,80 @@
+## MinIO CLI Task
+
+[MinIO](http://minio.io/) is a High Performance Object Storage released under Apache License v2.0.
+It is API compatible with Amazon S3 cloud storage service.
+
+## Purpose of this Task
+
+This task use the MinIO server [play.min.io](https://play.min.io) which is basically a playground for testing.
+
+This task can be used to interact with the MinIO server through a MinIO client.
+
+## Install the Task and create a secret
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/task/minio/0.1/minio-cli.yaml
```

Comment:
can we just rename the task dir from `task/minio` -> `task/minio-cli` or the YAML `task/minio/minio-cli.yaml` to `task/minio/minio.yaml`?